### DISCUSSION TITLE
Scheduled Upload Time

### DISCUSSION THREAD

**Discussion by bjk201** [lavinir/hassio-onedrive-backup#97] [Ideas]
Hello,

is there a possibility to set an upload Time?

Currently i create 2x per Day a full Backup, which has a size of about 7GB. 

I Sound like to upload the Backup Filets at Night, so that my Internet Connection Is Not overloaded during the day. 

Upload only between 22:00 - 06:00 clock. 

thanks :)

**Comment by lavinir** [comment]
https://github.com/lavinir/hassio-onedrive-backup#allowed-hours-optional

**Reply by bjk201** [reply]
Thanks :)
